src/design/brand.stories.mdx

```diff
@@ -14,3 +15,11 @@ Cloud Four logo in SVG format. The logo fill color can be modified via the CSS
 <Preview>
   <Story name="Logo">{brandLogo}</Story>
 </Preview>
+
+# Favicons
+
+Cloud Four favicon in SVG format with dark mode support via `prefers-color-scheme`. Toggle dark mode in system settings to view alternate color mode.
```

Comment:
Is there an opportunity to credit the CSS-Tricks article you found helpful somewhere in this documentation?